Cython/Shadow.py

```diff
@@ -461,3 +461,16 @@ def threadid(self):
 import sys
 sys.modules['cython.parallel'] = CythonDotParallel()
 del sys
+
+
+def __getattr__(name, module=__name__):
```

Comment:
Nice idea! Worth a docstring, though, to explain what this is used for and why it's done this way.